source/game_videos.md

```diff
@@ -0,0 +1,97 @@
+# Game Videos
+
+## NOTE: video links don't work in some browsers, if you have issues use chrome
```

Comment:
```suggestion
.. note::

    Linked videos may not play in some browsers, if you have issues use Chrome, Brave or Safari.
```

Reply:
e.g. here's a screenshot of my locally tested version:
<img width="813" alt="Screenshot 2024-10-19 at 1 29 23 PM" src="https://github.com/user-attachments/assets/e8b68295-c834-40db-b91c-65c7c5b81d90">

<img width="972" alt="Screenshot 2024-10-19 at 1 33 02 PM" src="https://github.com/user-attachments/assets/1cf260f3-a715-4bc6-9150-723e7f25da25">